content/_help/verify-your-identity/verify-your-address-by-mail._fr.md

```diff
@@ -10,17 +10,17 @@ redirect_from:
 La vérification d'identité est le processus par lequel vous prouvez que vous êtes vous-même - et non quelqu'un qui prétend être vous. L'une des étapes de ce processus exige que vous fournissiez votre numéro de téléphone principal pour vérifier votre identité.
 
 
-Si vous ne pouvez pas effectuer la vérification par téléphone, nous vous enverrons une lettre contenant un code à usage unique. Vous utiliserez ce code pour terminer la vérification de votre identité et la configuration de votre compte.
+Si vous ne pouvez pas effectuer la vérification par téléphone, nous vous enverrons une lettre contenant un code de vérification. Vous utiliserez ce code pour terminer la vérification de votre identité et la configuration de votre compte.
 
-La lettre est envoyée via le service postal américain trois à sept jours ouvrables après que vous ayez demandé le code.  Le code à usage unique expirera 30 jours après la date à laquelle vous avez envoyé votre demande.
+La lettre est envoyée via le service postal américain trois à sept jours ouvrables après que vous ayez demandé le code.  Le code de vérification expirera 30 jours après la date à laquelle vous avez envoyé votre demande.
```

Comment:
French phrasing for first class mail is slightly different in the app than here: "Les lettres sont envoyées le jour ouvrable suivant par le courrier de première classe d'USPS et arrivent dans un délai de 5 à 10 jours.

Reply:
Let's leave this one for now and we'll address it in the 5 to 10 days update.